File: Latihan3/Latihan3_Python/Disk.py
class Disk:
    # class attribute
    __capacity = ""
    __price = 0
    __type = ""

    # constructors
    def __init__(self, tipe="", capacity=0, price=0):
        self.__capacity = capacity
        self.__type = tipe
        self.__price = price

    # setter and getter
    def setType(self, tipe):
        self.__type = tipe

    def getType(self):
        return self.__type

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

File: Latihan3/Latihan3_Java/Disk.java
public class Disk {

	// class attribute
	String type;
	String capacity;
	int price;

	// constructors
	Disk() {
	}

	Disk(String type, String capacity, int price) {
		this.type = type;
		this.capacity = capacity;
		this.price = price;
	}

	// getter and setter
	public void setType(String type) {
		this.type = type;
	}

	public String getType() {
		return type;
	}

	public void setCapacity(String capacity) {
		this.capacity = capacity;
	}

	public String getCapacity() {
		return capacity;
	}

	public void setPrice(int price) {
		this.price = price;
	}

	public int getPrice() {
		return price;
	}

}
